src/Phpro/SoapClient/Soap/Driver/ExtSoap/ExtSoapOptions.php

```diff
@@ -114,6 +133,18 @@ public function disableWsdlCache(): self
         return $this;
     }
 
+    public function withMetaOptions(callable $manipulator): self
```

Comment:
Just to be consistent ?

```suggestion
    public function withMetadataOptions(callable $manipulator): self
```

Reply:
Fixed